app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,384 @@
+    <string name="login_activity_name" translatable="false">monerujo</string>
+    <string name="wallet_activity_name">Wallet</string>
+
+    <string name="menu_testnet">Testnet</string>
+    <string name="menu_about">Über</string>
+    <string name="menu_privacy">Datenschutzerklärung</string>
+
+    <string name="menu_share">Teilen</string>
+    <string name="menu_help">Hilfe</string>
+    <string name="menu_info">Details</string>
+    <string name="menu_receive">Empfangen</string>
+    <string name="menu_rename">Umbenennen &#8230;</string>
+    <string name="menu_archive">Archivieren</string>
+    <string name="menu_backup">Backup</string>
+    <string name="menu_changepw">Passwortphrase ändern</string>
+
+    <string name="password_weak">Tippe weiter &#8230;</string>
+    <string name="password_fair">Hmmm &#8230;</string>
+    <string name="password_good">Komm schon, das kannst du besser!</string>
+    <string name="password_strong">Fast da &#8230;</string>
+    <string name="password_very_strong">Yeah baby, h4x0r style!</string>
+
+    <string name="label_login_wallets">Wallets</string>
+    <string name="label_credits">Credits</string>
+    <string name="label_ok">OK</string>
+    <string name="label_cancel">Abbrechen</string>
+    <string name="label_close">Schließen</string>
+    <string name="label_wallet_advanced_details">Berühren für detaillierte Informationen</string>
+
+    <string name="label_send_success">Erfolgreich gesendet</string>
+    <string name="label_send_done">Fertig</string>
+
+    <string name="label_receive_info_gen_qr_code">Berühren für QR-Code</string>
+    <string name="info_send_prio_fees">Höhere Priorität = Höhere Gebühr</string>
+
+    <string name="info_xmrto_enabled">BTC Zahlung aktiviert, tippe für mehr Infos.</string>
+
+    <string name="info_xmrto"><![CDATA[
+        <b>Du hast eine BTC Adresse eingegeben.</b><br/>
+        <i>Du wirst XMR versenden und der Empfänger durch den <b>XMR.TO</b> Service BTC erhalten.</i>
+    ]]></string>
+
+    <string name="info_send_xmrto_success_order_label">XMR.TO Auftrag</string>
+
+    <string name="info_send_xmrto_success_sending">Sendet</string>
+    <string name="info_send_xmrto_success_status_label">Status:</string>
+    <string name="info_send_xmrto_success_btc">%1$s BTC</string>
+    <string name="info_send_xmrto_success_status">%1$s ist %1$s (#%1$d)</string>
+
+    <string name="info_txdetails_xmrto_key">Schlüssel:</string>
+    <string name="info_txdetails_xmrto_address">Adresse:</string>
+    <string name="info_txdetails_xmrto_status">Status:</string>
+    <string name="info_txdetails_xmrto_amount">%1$s BTC</string>
+
+
+    <string name="info_send_xmrto_paid">Bestätigung ausstehend</string>
+    <string name="info_send_xmrto_unpaid">Bezahlung ausstehend</string>
+    <string name="info_send_xmrto_error">XMR.TO Fehler (%1$s)</string>
+    <string name="info_send_xmrto_sent">BTC Gesendet!</string>
+    <string name="info_send_xmrto_query">Frage &#8230; ab</string>
```

Comment:
=> Frage ab &#8230;

Reply:
What will this resolve to? Is &#8230 the wallet name?